src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.css

```diff
@@ -12,12 +12,6 @@ table tr td {
     text-decoration: none;
 }
 
-.sub {
```

Comment:
What was this for?

Reply:
I've checked. I didn't see any use of this class.